ekm/signer_key_manager_test.go

```diff
@@ -788,3 +790,133 @@ func TestEkmListAccounts(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 2, len(accounts))
 }
+
+func TestConcurrentSlashingProtectionAttData(t *testing.T) {
+	require.NoError(t, bls.Init(bls.BLS12_381))
+
+	km := testKeyManager(t, nil)
```

Comment:
does this use Badger or in-memory storage?

we should make sure to use Badger because this is the risk that SSV adds to the EKM mix

Reply:
```go
km := testKeyManager(t, nil)
```
using
```go
func getBaseStorage(logger *zap.Logger) (basedb.Database, error) {
    return kv.NewInMemory(logger, basedb.Options{})
}
```